src/simulator/linear_trajectory.py

```diff
@@ -37,12 +37,17 @@ def __init__(self, anchors):
 
     def get_measurements(self):
         measurements = {}
+        realistic_outlier_injected = False
```

Comment:
I think we can remove this flag to keep it simple and allow the outlier to be injected possibly more than once in a single set of measurements. Can always lower the probability instead to lower the chances

Reply:
Keep to ensure single outlier injection to largely resemble actual data